src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageResponseCommentUiTest.java

```diff
@@ -21,152 +20,165 @@ protected void prepareTestData() {
     }
 
     @Test
-    public void feedbackResponseCommentActions() throws IOException {
-
-        InstructorFeedbackResultsPage resultsPage;
```

Comment:
Instead, use the original method and pass the variable to `addEmptyCommentToResponseAndCheckStatusMessage` etc.